Source/WebCore/platform/RectEdges.h

```diff
@@ -173,6 +175,23 @@ inline RectEdges<T>& operator+=(RectEdges<T>& a, const RectEdges<T>& b)
     return a;
 }
 
+template<typename T>
+void RectEdges<T>::unite(const RectEdges<T>& other)
+{
+    // Handle empty special cases first.
+    if (other.isZero())
+        return;
+    if (isZero()) {
+        *this = other;
+        return;
+    }
+
+    setTop(std::max(top(), other.top()));
+    setTop(std::max(right(), other.right()));
```

Comment:
setRight?

Reply:
I renamed it '`maximum` since it gets the maximum for all the four sides of `RectEdges`.